contracts/contracts/subnet/SubnetActorManagerFacet.sol

```diff
@@ -167,7 +167,7 @@ contract SubnetActorManagerFacet is SubnetActorModifiers, ReentrancyGuard, Pausa
             LibPower.setMetadataWithConfirm(msg.sender, publicKey);
             LibPower.depositWithConfirm(msg.sender, amount);
 
-            _patchGenesisValidators(msg.sender);
+            _increaseGenesisValidatorCollateral(msg.sender, amount);
```

Comment:
I think the underscore is not necessary. The function is marked as internal anyway.

Reply:
To my understanding `_` is a helper to disambiguate, no internal vs external at the call site more easily, but doesn't have a factual call limiting fact iiuc.